lib/circuit.js

```diff
@@ -187,6 +187,25 @@ class CircuitBreaker extends EventEmitter {
     }
   }
 
+  /**
+   * Shuts down this circuit breaker. All subsequent calls to the
+   * circuit will fail, returning a rejected promise.
+   */
+  shutdown () {
+    this.disable();
+    this.removeAllListeners();
+    this.hystrixStats.shutdown();
+    this[HYSTRIX_STATS] = undefined;
+  }
+
+  /**
+   * Determines if the circuit has been shutdown.
+   */
+  isShutdown () {
```

Comment:
A getter feels more appropriate given the rest of the attributes below are defined that way.